WordPress/pattern-directory
# Pattern Grid: Update design of pattern cards

Fixes #103 - This updates the UI of the pattern cards to match the new design:
- Title & meta are always visible, only adding favorite count for now since we don't have views
- Show small author avatar
- Show a status label on draft/pending/declined patterns (seen in My Patterns)
- Fix an issue with encoded characters in pattern titles (ex, `&amp;`)
- Fix extra space in the preview — should fix "Extra space on the left/right of full-width items" in #156 

❗  This uses a fake favorite value for now, to get the UI working. I think we'll need a solution with caching to get this value for real, so that discussion can happen in #100 

### Screenshots

Browsing patterns:
![all-patterns](https://user-images.githubusercontent.com/541093/121741417-82b57000-cacc-11eb-96d7-ca3e904bbd2e.png)

My patterns:
![my-patterns](https://user-images.githubusercontent.com/541093/121741418-83e69d00-cacc-11eb-8e3b-d194a3da1eca.png)

Responsive styles - drops to 2 column around 770px, and to 1 col around 525px.
![all-patterns-small](https://user-images.githubusercontent.com/541093/121741414-81844300-cacc-11eb-8b07-6d9a0fbe5ce8.png)

![all-patterns-mobile](https://user-images.githubusercontent.com/541093/121741410-80ebac80-cacc-11eb-8f30-878222796f59.png)


### How to test the changes in this Pull Request:

- Build the branch with `yarn workspaces run build`
- Make sure you have a page called My Patterns (see #152), and a handful of patterns by your user
- View the pattern archives (all and categories)
- They should match the figma design
- View My Patterns
- The patterns should match the design, draft & pending patterns should be labelled
- Resizing the screen shouldn't break anything
